store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -2072,4 +1797,35 @@ public void testReadingNullValues() {
     }
   }
 
+  @Test public void testSdkWriteWhenArrayOfStringIsEmpty()
+      throws IOException, InvalidLoadOptionException {
+
+    CarbonProperties.getInstance()
+        .addProperty(CarbonCommonConstants.CARBON_BAD_RECORDS_ACTION, "FAIL");
```

Comment:
same as above, take a backup of CARBON_BAD_RECORDS_ACTION and set it at the end of the test case